src/redux/store.js

```diff
@@ -3,14 +3,15 @@ import ReduxLogger from 'redux-logger';
 
 import rootReducer from './reducers';
 import localStorageUtil from '../utils/localStorage';
+import thunk from 'redux-thunk';
 
 const persistedState = localStorageUtil.loadState();
 
 const store = createStore(
   rootReducer,
   persistedState,
   compose(
-    applyMiddleware(ReduxLogger),
+    applyMiddleware(thunk, ReduxLogger),
     window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
   )
```

Comment:
Looks like we'll need advanced setup here:
https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup

There is even package for that: `redux-devtools-extension`